playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -101,18 +101,11 @@ EDXAPP_MARKETO_CLIENT_ID: ''
 EDXAPP_MARKETO_CLIENT_SECRET: ''
 EDXAPP_MARKETO_LIST_ID: 0
 EDXAPP_MONGODB_LOG:
-  host: "{{ EDXAPP_MONGO_HOSTS[0] }}"
+  host: "{{ EDXAPP_MONGO_HOSTS|join(',') }}"
```

Comment:
Note: most other places, besides here (elsewhere in edx's config, in our JSON, etc.) just reference this variable as a list.
This is the only place (besides the outdated comment above) that mentions keeping this as a comma-delimited string. At one point, edx actually had a big scary message about "THIS MUST BE A STRING", but they have since deleted that from their README.

I bet we could do the same here (keep it as a list), but I'm inclined to not change it for now, since this is how we've been running it in prod.